src/pages/createConfig/step2AddConfig/step2AddConfig.tsx

```diff
@@ -18,31 +18,30 @@ type Step2AddConfigProps = {
   initialJsonStringData?: string | undefined;
 };
 export const Step2AddConfig: React.FC<Step2AddConfigProps> = ({ onDataChange, onJsonStringChange, schemaId, initialJsonStringData }) => {
-  const fetchSchemaDereference = useCallback(() => getSchema({ id: schemaId, shouldDereference: true }), [schemaId]);
+  const fetchSchemaDereference = useCallback(() => getSchema({ id: schemaId, shouldDereference: false }), [schemaId]);
```

Comment:
```suggestion
  const fetchSchema = useCallback(() => getSchema({ id: schemaId, shouldDereference: false }), [schemaId]);
```
its not dereferenced anymore no?

Reply:
you are right, changed.